__tests__/app.test.js

```diff
@@ -375,3 +375,42 @@ describe("GET api/users", () => {
       });
   });
 });
+
+describe("QUERY TOPIC api/articles", () => {
+  test("GET:200 returns a filtered array of article objects with corresponding properties", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(articles.length).toBe(1);
+        expect(articles[0]).toMatchObject({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: expect.any(String),
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
+        });
```

Comment:
This is a great test, I wonder if with the fact that `cats` only has one article attached it might be better to test for a topic with multiple articles so that it's clearer it should return all articles for a given topic 🤔